int16 id
int8[] sequence
float32 center_x
float32 center_y
